test/regression/aarch64/Exception.cc

```diff
@@ -76,6 +119,13 @@ TEST_P(Exception, svcr) {
   )");
   CHECK_NEON(0, uint64_t, fillNeon<uint64_t>({0}, VL / 8));
 
+  RUN_AARCH64(R"(
+    # Ensure z regs get enabled when SM enabled
+    smstart sm
+    dup z0.d, #3
```

Comment:
Lines 124 and 125 are the other way around than the surrounding tests (i.e., smstart before dup).   Is this intended?